spec/axe.spec.js

```diff
@@ -14,6 +14,15 @@ const { fractalLoad } = require('./delayed-root-suite');
 const HOSTNAME = os.hostname().toLowerCase();
 const REMOTE_CHROME_URL = process.env[ 'REMOTE_CHROME_URL' ];
 const AXE_JS = fs.readFileSync(__dirname + '/../node_modules/axe-core/axe.js');
+const AXE_CONTEXT = JSON.stringify({
+  exclude: [
+    // For some reason aXe takes a lot longer if it needs to dive into
+    // iframes with data: URIs. The content of these iframes is just for
+    // non-USWDS example content anyways, so just skip them to speed things
+    // up.
+    [ 'iframe[src^="data:"]' ],
+  ],
+});
```

Comment:
For more details on this object, see https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#context-parameter.